app/controllers/gestionnaires/dossiers_controller.rb

```diff
@@ -93,6 +93,13 @@ def repasser_en_construction
       render partial: 'state_button_refresh', locals: { dossier: dossier }
     end
 
+    def repasser_en_instruction
+      dossier.repasser_en_instruction!(current_gestionnaire)
+      flash.notice = 'Dossier repassé en instruction.'
```

Comment:
Suggestion un peu plus fancy:
```suggestion
      flash.notice = "Le dossier #{dossier.id} a été repassé en instruction.'
```

(mais je constate que c’est la même formulation que dans les autres scenarios)